package.json

```diff
@@ -54,17 +55,22 @@
   },
   "license": "LicenseRef-LICENSE",
   "lint-staged": {
-    "*.{js,jsx}": [
-      "eslint --fix",
-      "git add"
-    ],
-    "*.ts": [
-      "tslint --config tslint.json --project tsconfig.json --fix",
-      "git add"
-    ],
-    "*.{json,md,css}": [
-      "prettier --write",
-      "git add"
+    "linters": {
+      "*.{js,jsx}": [
+        "eslint --fix",
+        "git add"
+      ],
+      "*.ts": [
+        "tslint --config tslint.json --project tsconfig.json --fix",
+        "git add"
+      ],
+      "*.{json,md,css}": [
+        "prettier --write",
+        "git add"
+      ]
+    },
+    "ignore": [
+      "electron/**/*.test*.ts"
```

Comment:
Why ignore them?

Reply:
Because the linter uses our main `tsconfig.json` when linting and the test files are excluded from our main `tsconfig.json` and the linter will then complain about that but the test files actually don't need to be included in our main `tsconfig.json` because they are piped through the Babel TypeScript compiler which has it's own compatibility configuration.

Reply:
Then let's create a `tsconfig.jasmine.json` so that `tsconfig` can fix them on commit.

Reply:
Seems like we can go without it.